### PR TITLE
Feature #29057: Add service_type support

### PR DESCRIPTION
**Pull request by AntonSizov**

### CODE REVIEW

File: src/smppload_lazy_messages_random.erl [hunk 1]
```diff
@@ -45,6 +46,7 @@ init(Config) ->
     DataCoding = ?ENCODING_SCHEME_LATIN_1,
     {MaxMsgLen, MaxSegLen} = smppload_utils:max_msg_seg(DataCoding),
     Length = ?gv(length, Config, MaxMsgLen),
+    ServiceType = ?gv(service_type, Config, MaxMsgLen),
```
Comment (ten0s) [inline src/smppload_lazy_messages_random.erl]: Why MaxMsgLen is the default value?

**Comment by ten0s** [comment]
Please also update the README file concerning both service_type and ssl

**Comment by AntonSizov** [comment]
@ten0s Fixed

**Comment by ten0s** [comment]
I think it would be right, at least for consistency sake, to add service_type to src/smppload_lazy_messages_file.erl as well. 

What do you think?

**Comment by AntonSizov** [comment]
@ten0s, Done.
Also I fixed badmatch that appeared during tests because source addr was undefined. It reproduces only in file mode AFAIK.

**Comment by ten0s** [comment]
10x

**Pull request merged**